lib/pid_ng/include/PID.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 202 COGIP Robotics association <cogip35@gmail.com>
```

Comment:
For `PID.hpp` location, you miss a subdir: `include/pid_ng/PID.hpp`.

Reply:
done